README.md

```diff
@@ -1,14 +1,44 @@
-# Terraform Docker Image
+# Terraform Docker Image [![Docker Repository on Quay.io](https://quay.io/repository/wantedly/terraform/status "Docker Repository on Quay.io")](https://quay.io/repository/wantedly/terraform)
 Docker Image for [Terraform](https://terraform.io).
 Please see [official document](https://terraform.io/docs/index.html) for more information about Terraform.
 
 ## SUPPORTED TAGS
 
-* TBD
+* `latest`
+ * Terraform 0.4.2
 
 ## HOW TO USE
 
-TBD
+```bash
+# Get the Docker image
+$ docker pull quay.io/wantedly/terraform
```

Comment:
`quay.io/wantedly/terraform:latest` is better :point_up: 
(And I know I do the same thing other repo...)

## WHY
As far as I know, If we don't specify tag, docker pulls all image (means all tagged images) on the repo.
Sometime this takes a long time.